src/main/java/seedu/malitio/model/ModelManager.java

```diff
@@ -162,6 +164,22 @@ public void editEvent(Event edited, ReadOnlyEvent beforeEdit) throws DuplicateEv
         indicatemalitioChanged();
     }
     
+	@Override
+	public void completeFloatingTask(ReadOnlyFloatingTask taskToComplete) throws FloatingTaskCompletedException, FloatingTaskNotFoundException {
+		malitio.completeTask(taskToComplete);
+        updateFilteredEventListToShowAll();
```

Comment:
Is there a reason you used Boolean instead of boolean in some classes?
not really, probably some typo